pandas/plotting/_core.py

```diff
@@ -2852,22 +2852,48 @@ def pie(self, y=None, **kwds):
 
     def scatter(self, x, y, s=None, c=None, **kwds):
         """
-        Scatter plot
+        A scatter plot with point size *s* and color *c*.
+
+        The coordinates of each point *x,y* are defined by two dataframe
+        columns and filled circles are used to represent each point.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
+        x : column name or column position
```

Comment:
I think `label or position` is OK. You can remove optional, as both `x` and `y` are required.

Reply:
But the description of the argument `c` specifies column name or column position. My intention was to make it consistent

Reply:
There is some confusion about using "label or position" as type. In other PR @datapythonista suggests using real types like "int or str" as the argument type, and mention it's the column name or position in the description of the parameter on next line.

Reply:
yes, my mistake, I realized later that `label` could be any type, I guess that's why `label or position` is being used.